src/state/apis/nft/nftApi.ts

```diff
@@ -276,6 +295,12 @@ export const nftApi = createApi({
             if (!item.collection.assetId) return acc
             const cachedCollection = selectNftCollectionById(state, item.collection.assetId)
             if (cachedCollection?.isSpam) item.collection.isSpam = true
+            if (
+              nftNameBlacklistRegex.test(
+                `${item.description.toLowerCase()} ${item.name.toLowerCase()} ${item.symbol.toLowerCase()}`,
+              )
+            )
```

Comment:
the rgx is case insensitive, so maybe something like this might be more readable?
```suggestion
            if ([item.description, item.name, item.symbol].some(value => nftNameBlacklistRegex.test(value)))
```

Reply:
1cd8565e9f894e822cabd20eac7c33bfea615e75